agg-ast/ast/src/definitions.rs

```diff
@@ -620,6 +622,23 @@ pub struct Bucket {
     pub default: Option<Bson>,
     pub output: Option<LinkedHashMap<String, Expression>>,
 }
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct RankFusion {
+    pub input: RankFusionPipeline,
+    pub combination: Option<RankFusionCombination>,
+    pub score_details: bool,
+}
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct RankFusionPipeline {
```

Comment:
nit: this is a tiny nit, but maybe something like `RankFusionInput` would make more sense, sense the input is a document of _multiple_ pipelines